hamocc/mo_param_bgc.F90

```diff
@@ -89,6 +90,12 @@ module mo_param_bgc
   real, parameter :: rdn2o1 = 2*ro2ut - 2.5*rnit  ! moles N2O used for remineralisation of 1 mole P
   real, parameter :: rdn2o2 = 2*ro2ut - 2*rnit    ! moles N2 released  for remineralisation of 1 mole P
 
+  ! Decay parameter for C14, HalfLive = 5700 years
+  real, parameter :: c14_t_half = 5700.*365.      ! Half life of 14C [days]
+
+  ! Scaling factor for pH dependency (used if with_dmsph=.true.)
```

Comment:
I would still put the DMS part into the DMS section below.... - just to have that all together.